src/query.js

```diff
@@ -427,6 +436,7 @@ export default class Query {
         ResTarget(ColumnRef('_updated_longitude'), 'updated_longitude'),
         ResTarget(ColumnRef('_updated_altitude'), 'updated_altitude'),
         ResTarget(ColumnRef('_updated_horizontal_accuracy'), 'updated_horizontal_accuracy'),
+        ...(!this.schema.hasRecordSeriesID ? [] : [ResTarget(ColumnRef('_record_series_id'), 'record_series_id')]),
```

Comment:
Would it make sense to use a variable like we're doing for `recordKeyColumns` and `joinedColumns`? That way it wouldn't be repeated on line 481.

Reply:
For better legibility, absolutely! Thanks

Reply:
@toddrun Oh.. I recall why now. Would you happen to know why the `recordKeyColumns` are separate from the `joinedColumns`? i.e., would it make sense to push them (as well as the `_record_series_id`) to joinedColumns instead of storing it in a variable?

Reply:
I believe I see why.. we don't add `recordKeyColumns` when there's a repeatableKey